docs/pages/identity-governance/device-trust/intune-integration.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Microsoft Intune Integration
+description: Sync your Microsoft Intune inventory into Teleport
+labels:
+ - how-to
+ - identity-governance
+---
+
+Device Trust Microsoft Intune integration lets you automatically sync your managed devices from
+Intune into Teleport.
+
+## How it works
+
+The Teleport Intune integration periodically reads your managed device inventory from Microsoft
+Intune and syncs it to Teleport. It performs both incremental (called "partial") and full syncs,
+as well as removals from Teleport if a device is removed from Intune.
+
+Syncing devices from Intune is an **inventory management** step, equivalent to
+automatically running the corresponding `tctl devices add` commands.
+
+See the [Device Trust guide](device-trust.mdx) for fundamental Device Trust concepts
+and behavior.
+
+<Admonition type="note">
+The Microsoft Intune integration is available exclusively as a hosted plugin through the
+Teleport Web UI. Running it as a standalone service is not currently supported.
+</Admonition>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx edition="Teleport Enterprise"!)
+
+(!docs/pages/includes/device-trust/prereqs.mdx!)
+- Access to Microsoft Entra ID with Global Administrator or Application Administrator permissions.
+- An active Microsoft Intune subscription with managed devices.
+
+## Step 1/3. Register an application in Microsoft Entra ID
+
+[Create and configure an application](https://learn.microsoft.com/en-us/graph/auth-register-app-v2)
```

Comment:
I would make it more explicit that this is a link to the Microsoft docs so the reader knows they are leaving the page, following another set of instructions, and coming back.

The same goes for the links in Step 2:
- "Grant the application permission"
- "create a client secret"

Reply:
This is tricky because those Microsoft docs have the same instructions as ours, pretty much. I left links to them so that in case ours become outdated, the user can double-check them against the official docs. I'll leave a more explicit message about it.